hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/trigger/Trigger.java

```diff
@@ -31,99 +31,189 @@
     @JsonInclude
     private EventType eventType;
 
+    @ApiModelProperty(value = "The category of the event produced by the trigger.",
+            position = 6)
     @JsonInclude
     private String eventCategory;
 
     /** Defaults to the Trigger Description if not null, otherwise the trigger name. */
     @JsonInclude
+    @ApiModelProperty(value = "The text of the event produced by the trigger.",
+            position = 7,
+            example = "If not eventText defined. Description will be used. If not description defined, trigger name " +
+                    "will be used.")
     private String eventText;
 
     // Ignored for Event Triggers
+    @ApiModelProperty(value = "Severity of a trigger.",
+            position = 8,
+            example = "MEDIUM")
     @JsonInclude
     private Severity severity;
 
+    @ApiModelProperty(value = "Properties defined by the user for this trigger. Context is propagated " +
+            "on generated Events/Alerts. Context cannot be used as criteria on finder methods.",
+            position = 9)
     @JsonInclude(Include.NON_EMPTY)
     protected Map<String, String> context;
 
+    @ApiModelProperty(value = "Tags defined by the user for this trigger. A tag is a [name, value] pair." +
+            "Tags can be used as criteria on finder methods. + \n" +
+            "Tag value cannot be null.",
+            position = 10)
     @JsonInclude(Include.NON_EMPTY)
     protected Map<String, String> tags;
 
     /** A list of links to actions represented by TriggerAction*/
+    @ApiModelProperty(value = "A list of links to actions.",
+            position = 11)
     @JsonInclude(Include.NON_EMPTY)
     private Set<TriggerAction> actions;
 
     /** Disable automatically after firing */
+    @ApiModelProperty(value = "Disable automatically after firing.",
+            position = 12,
+            example = "false")
     @JsonInclude
     private boolean autoDisable;
 
     /** Enable automatically if disabled and resolved manually */
+    @ApiModelProperty(value = "Enable automatically if disabled and resolved manually.",
+            position = 13,
+            example = "false")
     @JsonInclude
     private boolean autoEnable;
 
     /** Switch to auto-resolve mode after firing */
+    @ApiModelProperty(value = "Switch to auto-resolve mode after firing.",
+            position = 14,
+            example = "false")
     @JsonInclude
     private boolean autoResolve;
 
     /** Resolve all unresolved alerts when auto-resolve condition-set is satisfied */
+    @ApiModelProperty(value = "Resolve all unresolved alerts when auto-resolve condition-set is satisfied.",
+            position = 15,
+            example = "false")
     @JsonInclude
     private boolean autoResolveAlerts;
 
+    @ApiModelProperty(value = "The policy used for deciding whether the trigger auto-resolved condition-set is " +
+            "satisfied. Either ALL conditions must evaluate to true, or whether ANY one is enough.",
```

Comment:
Suggestion:
'Either ALL conditions must evaluate to true, or whether ANY one is enough.'
->
'ALL conditions must evaluate to true or ANY one condition must evaluate to true.'